content/post/practical-stegen.Rmd

```diff
@@ -0,0 +1,861 @@
+interviews to provide names of persons who attended the party.
+
+Overall, 291 responded to enquiries and 103 cases were identified. The linelist
+analysed in this case study was built from these 291 responses.
+
+
+## This case study
+
+In this case study, we will take you through the analysis of this epidemic. This
+will be the occasion to illustrate more generally useful practices for data
+analysis using **R**, including:
+
+- how to read data from Excel
+- how to explore data using tables and summaries
+- how to clean data
+- how to make graphics to describe the data
+- how to test if specific food items are linked to the disease
+
+
+
+
+
+# Initial data processing
+
+## Loading required packages
+
+The following packages will be used in the case study:
+
+- *here*: to find the path to data or script files
+- *readxl*: to read Excel spreadsheets into **R**
+- *incidence*: to build epicurves
+- *epitrix*: to clean labels from our spreadsheet
+- dplyr: to help with factors
+- ggplot2: to create custom visualisations
+
+If we have these packages installed, we can tell R to load these packages from
+our R library:
+
+```{r stegen_libraries, warning=FALSE, message=FALSE}
+library("here")      # find data/script files
+library("readxl")    # read xlsx files
+library("incidence") # make epicurves
+library("epitrix")   # clean labels and variables
+library("dplyr")     # general data handling
+library("ggplot2")   # advanced graphics
+```
+
+<details>
+<summary> <b> Package not installed? Did you get an error? </b> </summary>
+
+Note that you will get an error if the packages have not been installed on your
+system. To install them (you only need to do this once!), type:
+
+```{r stegen_install_libraries, eval = FALSE}
+install.packages("here")
+install.packages("readxl")
+install.packages("incidence")
+install.packages("epitrix")
+install.packages("dplyr")
+install.packages("ggplot2")
+```
+
+</details>
+
+Loading these packages makes all functions implemented by the packages
+available, so that `function_name()` can be used directly (without the
+`package_name::` prefix). This is not a problem here as these packages do not
+implement functions with identical names.
+
+
+
+## Importing data from Excel
+
+Linelist data can be read from various formats, including flat text files (e.g.
+`.txt`, `.csv`), other statistical software (e.g. STATA) or Excel spreadsheets.
+We illustrate the latter, which is probably the most common format.  We assume
+that [the data file `stegen_raw.xlsx`](../../data/stegen_raw.xlsx) has been
+saved in a `data/` folder of your project, and that your current R session is
+at the root of the project.
+
+
+Here we decompose the steps to read data in: finding the path to the data
+(`path_to_data`), using the function `read_xlsx()` from the *readxl* package to
+read data in, and saving the output in a new object `stegen`:
+
+```{r eval = FALSE}
+path_to_data <- here("data", "stegen_raw.xlsx")
```

Comment:
think here it is important to explicitly make them look at what "path_to_data" is. This way it might click that it just pulls together a pathway string for them...